app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/SearchViewModel.kt

```diff
@@ -175,7 +175,7 @@ class SearchViewModel @Inject constructor(
         val recentSearchesDao: RecentsDao = TcaDb.getInstance(context).recentsDao()
         val recentSearches: List<Recent> = recentSearchesDao.allRecentSearches.reversed() ?: emptyList()
         state.value = state.value.copy(
-            recentSearches = recentSearches
+            recentSearches = recentSearches.filter { x -> x.type != 1 } // filter for undesirable stations
```

Comment:
```suggestion
            recentSearches = recentSearches.filter { x -> x.type != STATIONS } // filter for undesirable stations
```
Please change this to the constant to make the code easier to understand. This is exactly the type you want to use. (And import `import de.tum.in.tumcampusapp.component.other.general.RecentsDao.STATIONS`)